### DISCUSSION TITLE
[Bug] Some BSSIDs appear in WiFi Explorer Pro, then disappear

### DISCUSSION THREAD

**Discussion by Coder84619** [WLAN-Pi/feedback#65] [General Inquiries] [answered]
I'm using Wi-Fi Explorer Pro and the wlan Pi R4 (3.2.0), with the provided Comfast AX1800 USB dongle. I have three Ruckus R770 APs (Wi-Fi 7 with tri-band) broadcasting multiple SSIDs on each band. "Most" of the time when using Wi-Fi Explorer Pro and the wlan Pi as a remote sensor it will see all three BSSIDs for all my SSIDs. However, at seemingly random intervals it will go 'blind' to one or more APs and the associated BSSIDs. For example, tonight it was showing 3 three BSSIDs for a good 30-45 minutes, then all of a sudden one AP's worth of BSSIDs dropped out. I rebooted the wlan Pi, quit/restarted wifi-explorer pro (Mac) to no avail. I know the AP is still broadcasting, as I ran Airport utility on my iPhone and verified 3 BSSIDs for each SSID. 

I thought I opened an issue about this a while back, but I can't find it. Sorry for the dupe if I made one.

**Comment by joshschmelzle** [comment]
@adriangranados for ideas.

**Reply by adriangranados** [reply, marked as answer]
I recommend trying again with the latest version of WiFi Explorer Pro 3 or WiFi Explorer Pro for Windows, as recent updates include major improvements and bug fixes related to the parsing of the Multiple BSSID IE and handling of nontransmitted BSSID profiles, which I suspect are being used in this scenario given the use of Wi-Fi 7 and 6 GHz.
